src/Stream.php

```diff
@@ -44,6 +44,13 @@ class Stream implements SeekableIterator
     protected $stream;
 
     /**
+     * Tell whether the stream support seek operations
+     *
+     * @var bool
+     */
+    protected $seekable;
```

Comment:
I would prefer the property to be called is_seekable since it will hold a boolean value.